src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -47,10 +47,17 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        Set<String> keywordsInLowerCase = new HashSet<>();
+        for (String keyword : keywords) {
+            keywordsInLowerCase.add(keyword.toLowerCase());
+        }
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInName = new HashSet<>();
+            for (String word : person.getName().getWordsInName()) {
+                wordsInName.add(word.toLowerCase());
```

Comment:
Maybe instead of looping through the getWordsInName method, why not change the method itself to ignore casing?